### ISSUE TITLE
Subtitles do not work

### ISSUE THREAD

**Issue by vikingnope** [prayag17/Blink#401]
**Describe the bug**
Subtitles work on Jellyfin Media Player but not on Blink

**To Reproduce**
1. Watch a show and try to turn on subtitles

**Expected behavior**
Subtitles should work

**Screenshots**

Jellyfin Media Player:

![image](https://github.com/user-attachments/assets/a7ff0959-cd4d-4a66-b78f-5454171c7d44)

Blink:
https://github.com/user-attachments/assets/ea0b7313-a72b-42fa-ab2e-2e6878f50737

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version Latest nightly build (a1bb8f8e236a211fef875803e3ed4c924e161411)
 - Jellyfin Version: 10.9.11
**Additional context**

It seems to be found just not working:

![image](https://github.com/user-attachments/assets/7fdf98f2-8de1-4ac5-911e-29af44cf07d5)

**Comment by Saturn745** [comment]
Duplicate of #394?

**Comment by vikingnope** [comment]
No, I dont think so, I do not even get the options to choose my subtitles, I just get nothing at all

**Comment by kloon15** [comment]
+1, i see the list of subs but nothing displayed no matter the format.
